pkgs/by-name/gu/guitarix/package.nix

```diff
@@ -0,0 +1,156 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  avahi,
+  bluez,
+  boost,
+  curl,
+  eigen,
+  faust,
+  fftwSinglePrec,
+  gettext,
+  glib,
+  glib-networking,
+  glibmm,
+  gperf,
+  gtk3,
+  gtkmm3,
+  hicolor-icon-theme,
+  intltool,
+  ladspaH,
+  libjack2,
+  liblo,
+  libsndfile,
+  lilv,
+  lrdf,
+  lv2,
+  pkg-config,
+  python3,
+  sassc,
+  wafHook,
+  which,
+  wrapGAppsHook3,
+  zita-convolver,
+  zita-resampler,
+
+  enableFaust ? !stdenv.hostPlatform.isLinux, # Transpiles Faust DSP code to C++
```

Comment:
`platform` is set to linux.  The default is always `false`?

Reply:
Yes. Theoretically, guitarix works on darwin and BSD. But, I haven't been able to get it building so the platform is set to linux. If someone were to get it working, this would become relevant. Basically, this is anticipating an expansion of the target platforms.